scenario/askhubadvisor.go

```diff
@@ -71,6 +71,7 @@ type (
 		SaveImages        bool                   `json:"saveimages" displayname:"Save images" doc-key:"askhubadvisor.saveimages"`
 		SaveImageFile     session.SyncedTemplate `json:"saveimagefile" displayname:"File name (without suffix)" doc-key:"askhubadvisor.saveimagefile" displayelement:"savefile"`
 		ThinkTimeSettings *ThinkTimeSettings     `json:"thinktime,omitempty" displayname:"Think time settings" doc-key:"askhubadvisor.thinktime"`
+		FollowupTypes     []followupType         `json:"followuptypes,omitempty" displayname:"Followup query type priority" doc-key:"askhubadvisor.followuptypes"`
```

Comment:
Not sure how this will play with QSEST? An array of a custom type

Reply:
Is an array of enummaps ok with the GUI?

Reply:
No, it will require some kind of new form.. I guess equivalent to current logic for strings or numbers, but with dropdowns for selecting the value.